vc/status-list/.htaccess

```diff
@@ -0,0 +1,6 @@
+Header set Access-Control-Allow-Origin *
+Header set Access-Control-Allow-Headers DNT,X-Mx-ReqToken,Keep-Alive,User-Agent,X-Requested-With,If-Modified$
+Options +FollowSymLinks
+RewriteEngine on
+RewriteRule ^$ https://w3c-ccg.github.io/vc-status-list-2021/ [R=302,L]
+RewriteRule ^v1$ https://w3c-ccg.github.io/vc-status-list-2021/contexts/v1.jsonld [R=302,L]
```

Comment:
This isn't working.

Reply:
@msporny When you get a chance, please see if this can be fixed.  It looks like the context was moved but it didn't get published.

Reply:
Works now.